test/gen_NISTKAT.c

```diff
@@ -13,12 +13,12 @@
 
 
 
-#if DILITHIUM_MODE == 2
-#define CRYPTO_ALGNAME "Dilithium2"
-#elif DILITHIUM_MODE == 3
-#define CRYPTO_ALGNAME "Dilithium3"
-#elif DILITHIUM_MODE == 5
-#define CRYPTO_ALGNAME "Dilithium5"
+#if MLD_MODE == 2
+#define CRYPTO_ALGNAME "ML-DSA-44"
+#elif MLD_MODE == 3
+#define CRYPTO_ALGNAME "ML-DSA-65"
+#elif MLD_MODE == 5
+#define CRYPTO_ALGNAME "ML-DSA-87"
 #endif
```

Comment:
Oh... this shouldn't work. I opened https://github.com/pq-code-package/mldsa-native/issues/61 for this.

Reply:
Addressed in https://github.com/pq-code-package/mldsa-native/pull/69/files#diff-b467b00ff5e52a98e2d2cb2d46247325542d8e0e46a66d6928aa8d04651fca18R16-R21. Left the names the same.